job/replay_manager.go

```diff
@@ -90,6 +109,117 @@ func (m *Manager) Replay(reqInput *models.ReplayWorkerRequest) (string, error) {
 	}
 }
 
+func (m *Manager) validate(ctx context.Context, replaySpecRepo store.ReplaySpecRepository, reqInput *models.ReplayWorkerRequest) error {
+	reqReplayTree, err := prepareTree(reqInput)
+	if err != nil {
+		return err
+	}
+	reqReplayNodes := reqReplayTree.GetAllNodes()
+
+	if !reqInput.Force {
```

Comment:
@mauliksoneji i dont think changing this to 'if reqInput.Force' will be better, because we have to put 'cancelConflictedReplays' calls in 2 places. thoughts?